### PR TITLE
Readme Update

### PR DESCRIPTION
**Pull request by NBumgardner**
This pull request **polishes the main readme**. My fork has a [preview of the PR](https://github.com/NBumgardner/Floe/tree/readme-update).
- Rewords two sections (_How do I get set up?_ and _Who do I talk to?_) for flow and conciseness.
- Added my name to the contacts, with space below for newcomers.
- Adds links to contributors and demos (`SavedBinFiles` folder) for navigation.
- Adds backquotes around all code snippets,  commands, and folders; for clarity and consistency.
- Changes level 3 headings to level 2, since only two heading levels are used.

Minor, consistent punctuation fixes include:
- Italicizes version number.
- Bolds debug warning instead of using all-caps.
- Minor punctuation fixes (for periods and colons).

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -2,46 +2,51 @@
 
 This is an action-puzzle game created by a team of students in Missouri State University's Computer Science department.
 
-### Overview
+## Overview
 
 Help Floe, the spirit of ice, bring winter to the land by freezing over lakes. There are a number of different enemies that stand in your path, however, and they aren't happy about Floe's intrusion on their territory. 
-Version 1.0
+*Version 1.0*
```
Comment (SamBumgardner) [inline readme.md:9]: If you could move the version down by another line (so it's not just at the end of the previous sentence) that'd be great!

File: readme.md [hunk 2]
```diff
@@ -2,46 +2,51 @@
...
+## Useful Notes
 
 * All controls are keyboard-based. 
  * Use the arrow keys to change menu selections and move Floe.
  * Use spacebar/enter to interact with the menu.
  * Anything else should be pretty self-explanatory.
 * Freeze all water tiles in the level to move on to the next.
 * Avoid all enemies.
-* DEBUG MODE: use the 'd' key to damage yourself, use 'n' to advance to the next level.
+* **Debug Mode:** Use the 'd' key to damage yourself, use 'n' to advance to the next level.
```
Comment (SamBumgardner) [inline readme.md]: What do you think of putting 'd' and 'n' in backquotes, so they're more clearly related to keyboard keys?

Ex: 
- **Debug Mode:** Use the `d` key to damage yourself, use `n` to advance to the next level.

File: readme.md [hunk 3]
```diff
@@ -2,46 +2,51 @@
...
-* The executable is located in /Floe/Floe/bin/flash/bin/
-* To run in debug mode, run the command "lime test -debug flash". Access the debug console with the backquote key.
+To build and run Floe, follow the steps below:
 
-### Useful Notes:
+ 1. Ensure you have [Haxe](http://www.haxe.org/download), [Haxepunk](http://www.haxepunk.com), and [OpenFL](http://www.openfl.org/download/) installed on your computer.
+ 2. Open a command prompt in `/Floe/Floe/`.
+ 3. Run the command `lime test flash` to build and run the executable.
+
+* The executable is located in `/Floe/Floe/bin/flash/bin/`.
```
Comment (SamBumgardner) [inline readme.md]: I don't know if I like having a bullet-point list immediately following a numbered list. What do you think of making these two points nested inside the above list?

**Review by SamBumgardner — COMMENTED** [review]
Overall, looks like a nice pull request. I left a few comments detailing some possible changes; please let me know if you think they're worthwhile.

File: readme.md (hunk 1 above)
Comment (NBumgardner) [reply readme.md:9]: I'm on it.

File: readme.md (hunk 3 above)
Comment (NBumgardner) [reply readme.md]: Do you want the bullets to become nested letters, nested bullets, or a following paragraph? I'll try them myself too.

File: readme.md (hunk 2 above)
Comment (NBumgardner) [reply readme.md]: I'm on it.

**Comment by NBumgardner** [comment]
This current PR has all three changes requested by @SamBumgardner so far.

**Future Notes:**
- I **prefer** placing the **version number inline** within the first two header sections. It would keep the readme shorter, and I think most readers will skip the version number.
- The **version number** may need to be **increased** to `1.0.1` or `1.1`, since this would be the first PR made since `1.0`, but no coding changes occurred.

**Comment by SamBumgardner** [comment]
Since this doesn't alter the project in any way, I don't think the version number should be incremented. We can change it later if we need to. 

I do prefer having the version number on its own line, because it's a separate subject from the lines before it. The rest of the changes look good to me as well, so I'll merge this in now, unless you have any objections.

**Comment by NBumgardner** [comment]
Thanks for the feedback. It is ready to be merged.

**Pull request merged**